core/templates/dev/head/pages/topic_editor/main_editor/topic_editor_tab_directive.html

```diff
@@ -47,6 +47,9 @@ <h4 ng-if="topic.getCanonicalStoryIds().length === 0" style="color: gray;">
 </div>
 
 <style>
+  .topic-editor textarea {
+    resize:vertical;
```

Comment:
add space after colon

/cc @oppia/dev-workflow-team this should be a lint check?

Reply:
Sorry for the delay. 
Filed #6494.
Thanks.

Reply:
@anubhavsinha98, We have merged this PR without this fix and now the tests are failing in develop branch!